lib/sidekiq-status/client_middleware.rb

```diff
@@ -35,5 +37,13 @@ def display_args(msg, queue)
       # For Sidekiq ~> 2.7
       return msg['args'].to_a.empty? ? nil : msg['args'].to_json
     end
+
+    def record_initial_status?(klass)
+      klass = Object.const_get(klass) if klass.is_a?(String)
+      return true unless klass.respond_to?(:record_initial_status?)
+      klass.record_initial_status?
+    rescue NameError
+      true
```

Comment:
Let's delete this `rescue` unless we can build a spec to justify its usefulness

Reply:
from a general perspective, the Worker class doesn't necessarily have to be available at the layer where we launch the job, just where we execute it (this is why strings are an acceptable input vs classes)

Reply:
e.g. http://coderascal.com/ruby/using-sidekiq-across-different-applications/

Reply:
I don't think we currently take advantage of this, but there is no reason to break such functionality

Reply:
The reason to not have this in is to write as little code as possible, so understanding it is as simple as possible. We don't use Sidekiq across different apps so writing code for that possibility is just noise.